app/Jobs/SendSignupToPhoenix.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Rogue\Jobs;
+
+use Rogue\Models\Signup;
+use Rogue\Services\Phoenix;
+use Rogue\Services\Registrar;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class SendSignupToPhoenix extends Job implements ShouldQueue
+{
+    use InteractsWithQueue, SerializesModels;
+
+    protected $signup;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Signup $signup)
+    {
+        $this->signup = $signup;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $phoenix = new Phoenix;
```

Comment:
These could both be injected from the service container using type-hinting on the `handle` method:

```php
/**
 * Execute the job.
 *
 * @return void
 */
public function handle(Phoenix $phoenix, Registrar $registrar)
{
    $drupal_id = $registrar->find($this->signup->northstar_id)->drupal_id;
    // ...
```

It doesn't make a difference right now since neither of those classes have their own injected dependencies, but it'll makes things easier later on down the road if you ever do. 😄